src/utils/constants.js

```diff
@@ -1,6 +1,6 @@
 export const BASE_URL = 'http://localhost:3000';
-export const BACKEND_URL = 'https://tesla-ah-staging.herokuapp.com';
-export const API_URL = 'https://tesla-ah-staging.herokuapp.com/api';
+export const API_URL = 'http://localhost:3000/api';
```

Comment:
You can't use localhost.

Reply:
what do you mean?

Reply:
The localhost url will not work when we deploy. Please update it.